src/porepy/grids/structured.py

```diff
@@ -148,10 +147,10 @@ def _create_2d_grid(
         num_faces = num_faces
         num_nodes = num_nodes
 
-        x_coord, y_coord = sp.meshgrid(nodes_x, nodes_y)
+        x_coord, y_coord = np.meshgrid(nodes_x, nodes_y)
 
         nodes = np.vstack(
-            (x_coord.flatten(), y_coord.flatten(), np.zeros(x_coord.size))
+            (x_coord.ravel(order="C"), y_coord.ravel(order="C"), np.zeros(x_coord.size))
```

Comment:
I assume you're sure about the order?

Reply:
Yes (it could even be that C-order is default, but C is correct).